libraries/bot-connector/src/test/java/com/microsoft/bot/connector/BotFrameworkAuthenticationFactoryTests.java

```diff
@@ -6,23 +6,30 @@
 import com.microsoft.bot.connector.authentication.AuthenticationConstants;
 import com.microsoft.bot.connector.authentication.BotFrameworkAuthentication;
 import com.microsoft.bot.connector.authentication.BotFrameworkAuthenticationFactory;
+import com.microsoft.bot.connector.authentication.ClaimsIdentity;
+import com.microsoft.bot.connector.authentication.ConnectorFactory;
 import com.microsoft.bot.connector.authentication.GovernmentAuthenticationConstants;
 import com.microsoft.bot.connector.authentication.PasswordServiceClientCredentialFactory;
+import com.microsoft.bot.connector.authentication.SkillValidation;
+import com.microsoft.bot.connector.authentication.UserTokenClient;
+import org.hamcrest.MatcherAssert;
 import org.junit.Assert;
 import org.junit.Test;
 
+import java.util.HashMap;
+
 import static org.hamcrest.CoreMatchers.instanceOf;
 
 public class BotFrameworkAuthenticationFactoryTests {
     @Test
     public void shouldCreateAnonymousBotFrameworkAuthentication() {
-        BotFrameworkAuthentication botFrameworkAuthentication = BotFrameworkAuthenticationFactory.create();
-        Assert.assertThat(botFrameworkAuthentication, instanceOf(BotFrameworkAuthentication.class));
+        BotFrameworkAuthentication bfA = BotFrameworkAuthenticationFactory.create();
```

Comment:
I'd rather that we respected the way variables are named in Java, but that's ok

Reply:
Yeah, I agree with you, I was confused to follow the name between JS and Java.. but both are `lowerCamelCase`, so it's ok I guess